comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -294,6 +294,14 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 		err = rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC)
 	}
 
+	streamLogsSource := config.Datadog().GetSource("enable_stream_logs")
```

Comment:
Would it make sense to unify spelling of `stream_logs` vs `streamlogs`?

Do we need to register this setting in `setup/config.go`?

Would it make sense to add these settings to the `config_template.yaml`?